pkgs/by-name/di/discover-overlay/package.nix

```
@@ -0,0 +1,42 @@
+{ lib, python3, fetchFromGitHub, gtk3, gobject-introspection, wrapGAppsHook }:
+python3.pkgs.buildPythonApplication rec {
+  pname = "discover-overlay";
+  version = "0.7.0";
+
+  src = fetchFromGitHub {
+    owner = "trigg";
+    repo = "Discover";
+    rev = "v${version}";
```

```suggestion
    rev = "refs/tags/v${version}";
```

prevents confusion if an identically-named branch appears